src/containers/shared/css/table.scss

```diff
@@ -20,7 +20,8 @@ table.basic {
     border-bottom: 1px solid $gray-800;
   }
 
-  td, th {
+  td,
+  th {
```

Comment:
it always feels weird to me to target elements by type. also wouldn't this apply to all `td` and `th` since the scss is minified into a single file and this is an SPA? @ckniffen

Reply:
Usually when it doesnt is calling out span or div.  In the case of tables it completely does especially in the context of a style meant to be used many places

Reply:
not following your response @ckniffen

Reply:
Styling tables using tbody, th, td, tr, tfoot, thead are all good ways to style elements.